_posts/2015-09-15-rnd-day-2015.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: post
+title: "R&D Day 2015 in the spotlight"
+date: "2015-09-15 08:10:00"
+icon: search-plus
+categories: tech-stories
+authors: ["Birgit"]
+---
+
+ePages developers from all company locations convened in Hamburg to spend an amazing time together on software development topics!
+It was all about interesting lectures and workshops, healthy discussions and knowledge transfer among different teams.
+
+Curtains up for these fantastic topics:
+
+# Spotlight: ePages software architecture
+
+Team Black started the R&D Day presenting their approach on a new software architecture that should be the technical foundation for ePages in the years to come.
+
+{% image blog-rnd-architecture.jpg %}{% endimage %}
+
+The following principles drive the teams' work:
+
+* support development on the product in teams distributed across epages locations
+* create an architecture that is easy to change and to maintain
+* focus on the Java ecosystem.
+
+Each software development team should be able to independently deliver a piece of software that is ready for production deployment.
+In addition, it should be possible to quickly change parts of the system without creating side effects.
+In order to achieve that, they split up the product in business domains, where each domain is implemented in a vertical that:
+
+* represents a deployable unit
+* is exclusively responsible for its functionality
+* owns its data
+* provides REST API to leverage access to its functions
+* communicates with other verticals only by using REST-based interfaces or events
+* its micro-architecture needs to follow some high-level guidelines to ensure it is operational within the macro-architecture.
+
+# Spotlight: App & Theme Store
+
+Team Red presented the architecture of the App & Theme Store.
+The Store allows merchants to customise the design of their shop with certain themes.
+Furthermore, they can add additional functionalities to their shop by installing apps developed by external developers using the ePages APIs.
+
+{% image blog-rnd-appstore.jpg %}{% endimage %}
+
+They explained the different app types, the store currently supports:
+
+* REST apps that use the ePages [REST API](https://developer.epages.com/)
+* SOAP apps that use the ePages [SOAP API](https://developer.epages.com/soap/index.html)
+* Themes and Affiliates
+
+The challenge was to provide an architecture which not only allows external developers to develop and submit their app easily, but also to provide an administration tool for the product management which allows them to distribute the apps to the different providers.
+
+Despite the brief time frame of the session and the complexity of the architecture, Team Red succeeded
+to present in an understandable way using several diagrams and live demos of the different App & Theme Store use cases:
```

Comment:
I would change to:
Team Red succeeded and presented, in an understandable way, several diagrams and live demos of the different App & Theme Store use cases: